distribution/packages/src/common/scripts/postinst

```diff
@@ -49,20 +49,68 @@ case "$1" in
         exit 1
     ;;
 esac
-
 # to pick up /usr/lib/sysctl.d/elasticsearch.conf
 if command -v systemctl > /dev/null; then
     systemctl restart systemd-sysctl.service || true
 fi
-
 if [ "x$IS_UPGRADE" != "xtrue" ]; then
+    # Don't exit immediately on error, we want to hopefully print some helpful banners
+    set +e
+    # Attempt to auto-configure security, this seems to be an installation
+    if ES_MAIN_CLASS=org.elasticsearch.xpack.security.cli.ConfigInitialNode \
+    ES_ADDITIONAL_SOURCES="x-pack-env;x-pack-security-env" \
+    ES_ADDITIONAL_CLASSPATH_DIRECTORIES=lib/tools/security-cli \
+    /usr/share/elasticsearch/bin/elasticsearch-cli -strict <<< ""; then
+        # Above command runs as root and TLS keystores are created group-owned by root. It's simple to correct the ownership here
+        for dir in "${ES_PATH_CONF}"/auto_config_on*
+        do
+            chown root:elasticsearch "${dir}"/http_keystore_local_node.p12
+            chown root:elasticsearch "${dir}"/http_ca.crt
+            chown root:elasticsearch "${dir}"/transport_keystore_all_nodes.p12
+        done
+        if INITIAL_PASSWORD=$(ES_MAIN_CLASS=org.elasticsearch.xpack.security.enrollment.tool.AutoConfigGenerateElasticPasswordHash \
+        ES_ADDITIONAL_SOURCES="x-pack-env;x-pack-security-env" \
+        ES_ADDITIONAL_CLASSPATH_DIRECTORIES=lib/tools/security-cli \
+        /usr/share/elasticsearch/bin/elasticsearch-cli); then
+            echo "##########         Security autoconfiguration information         ############"
+            echo "#                                                                            #"
+            echo "# Authentication and Authorization are enabled.                              #"
+            echo "# TLS for the transport and the http layers is enabled and configured.       #"
+            echo "#                                                                            #"
+            echo "# The password of the elastic superuser will be set to: ${INITIAL_PASSWORD} #"
+            echo "# upon starting elasticsearch for the first time                             #"
+            echo "#                                                                            #"
+            echo "##############################################################################"
```

Comment:
```suggestion
            cat <<EOF
+----------------------------------------------------------------------------+
|                  Security auto-configuration information                   |
+----------------------------------------------------------------------------+
|                                                                            |
| Authentication and Authorization are enabled.                              |
| TLS for the transport and the http layers is enabled and configured.       |
|                                                                            |
| The password of the elastic superuser will be set to: ${INITIAL_PASSWORD}  |
| upon starting elasticsearch for the first time                             |
|                                                                            |
+----------------------------------------------------------------------------+
EOF
```

Reply:
I suppose we could do something clever with `printf` if we want the box to always render nicely.

Reply:
You mean no matter what the size of the terminal is? I also looked into it a little bit but my lack of experience with bash was prominent. I can look into this more if folks agree that this is important from a UX perspective. I would argue that can agree to do this in a followup that doesn't need to be in place for 8.0.0-alpha2 but no disagreement about putting in the effort!

Reply:
No, I just meant keeping the RHS character aligned when there's variables in the output.

Reply:
Gotcha. The value of the output is always 20 chars so that _should_ always align without additional handling